src/page.ts

```diff
@@ -61,6 +61,8 @@ export type WaitForNetworkIdleOptions = {
 /** The options for sandboxing */
 export type SandboxOptions = {
   sandbox?: boolean;
+  // temporary, might not want to keep this here
+  userAgent?: string;
```

Comment:
Can we move this out of the sandbox options into a type that extends sandbox options? Alternatively, could you rename this to like page options or something?

Reply:
Yeah, I actually forgot to even remove this comment XD

I'll rearrange this later